src/js/legacy/mega-menu.js

```diff
@@ -30,6 +38,42 @@ class MegaMenu {
     }
   }
 
+  showMegaMenu() {
+    this.openControl.setAttribute('hidden', 'hidden');
+    this.menu.removeAttribute('hidden');
+    this.closeControl.removeAttribute('hidden');
+    if (!isWideScreen()) {
+      this.firstMenuElement.focus();
+    }
+  }
+
+  hideMegaMenu() {
+    this.closeControl.setAttribute('hidden', 'hidden');
+    this.menu.setAttribute('hidden', 'hidden');
+    this.openControl.removeAttribute('hidden');
+    this.menu.classList.remove('vetnav--submenu-expanded');
+  }
+
+  toggleSmallMegaMenu(e) {
+    if (!isWideScreen() && isEscape(e)) {
+      this.hideMegaMenu();
+      this.openControl.focus();
+    }
+  }
+
+  enterSmallMegaMenu(e) {
+    if (!isWideScreen() && isTab(e) && !e.shiftKey) {
+      e.preventDefault();
+      this.firstMenuElement.focus();
+    }
+  }
+
+  exitSmallMegaMenu(e) {
+    if (!isWideScreen() && isTab(e) && !e.shiftKey) {
+      this.lastTabbableElement.focus();
+    }
+  }
+
   showMegaMenu() {
```

Comment:
Is this duplicate code now?

Reply:
These are both (showMegaMenu/hideMegaMenu) the result of a bad merge, and will be removed.